### PR TITLE
Add Tencent Kona JDKs

### PR DESCRIPTION
**Pull request by johnshajiang**
Tencent Kona JDK is a downstream of OpenJDK.
It has three releases: [8], [11] and [17].

[8]:
<https://github.com/Tencent/TencentKona-8>

[11]:
<https://github.com/Tencent/TencentKona-11>

[17]:
<https://github.com/Tencent/TencentKona-17>

### CODE REVIEW

**Review by eddumelendez — APPROVED** [review]

**Comment by johnshajiang** [comment]
@eddumelendez 
Should the patch be merged?
Or any other jobs I should do?

File: src/main/java/io/sdkman/automigration/adapters/PackageAdapter.java [hunk 1]
```diff
@@ -25,6 +25,7 @@ public class PackageAdapter {
 			entry("graalvm_ce11", "grl"),
 			entry("graalvm_ce17", "grl"),
 			entry("graalvm_ce19", "grl"),
+            entry("kona", "tencent"),
```
Comment (eddumelendez) [inline src/main/java/io/sdkman/automigration/adapters/PackageAdapter.java:28]:
```suggestion
            entry("kona", "kona"),
```

**Review by eddumelendez — APPROVED** [review]

**Comment by eddumelendez** [comment]
Thanks for your contribution, @johnshajiang ! and sorry for the delay! Kona just landed to SDKman :D

Screenshot 2023-06-24 at 16 12 44

**Comment by johnshajiang** [comment]
@eddumelendez 
Thanks very much!
I'll have some tries.

Thanks again!

**Pull request merged**